### ISSUE TITLE
Confirmation prior to shutdown

### ISSUE THREAD

**Issue by romildo** [streetturtle/awesome-wm-widgets#240]
It would be very nice and more secure to have a widget asking for confirmation prior to taking the actions in the logout widget.

**Comment by streetturtle** [comment]
Agree, I'll check what would be the best way to do it. I think the easiest will be to create a confirmation popup with zenity, but ideally it should be a popup widget.
